Yoast/YoastSEO.js
# Changed snippet preview to show mobile preview by default

## Summary

This PR can be summarized in the following changelog entry:
Snippet preview now shows mobile preview by default.

## Test instructions

This PR can be tested by following these steps:

* Link YoastSEO.js to `wordpress-seo` using `yarn link` and build the project.
* Check various pages that contain the snippet preview and verify mobile is selected by default.

Fixes Yoast/wordpress-seo#7533
